src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -525,6 +529,11 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
    * <p>All empty feedback response will be deleted; For non-empty responses, update/create them if necessary.
    */
   saveFeedbackResponses(): void {
+    if (this.submissionForm && !this.submissionForm.isValidSubmission) {
+      this.statusMessageService.showErrorToast(
+          'A distribute points question has an invalid response, please check your submission again');
```

Comment:
I think there is no need to show error-specific message here (as they have been shown in the constraint). Just disable the submission button and say "please check the constrains" or something.